bower.json

```diff
@@ -42,6 +42,7 @@
     "topojson": "~1.6.19",
     "dagre": "https://raw.githubusercontent.com/cpettitt/dagre/v0.7.3/dist/dagre.js",
     "d3-cloud": "~1.2.1",
+    "d3-plugins": "*",
```

Comment:
No - it means pull in latest version.  Because the plugins don't have semantic versioning, we are stuck with using the latest (which is what * is).
FYI - we do pull down all the plugins (but they are all small and self contained, but we don't pull them into the library because they are there, only the ones which are used get auto-included.